apps/server/src/modules/task/controller/task.controller.ts

```diff
@@ -95,4 +95,37 @@ export class TaskController {
 		const dto = CopyMapper.mapToResponse(copyStatus);
 		return dto;
 	}
+
+	@Get(':taskId')
+	async find(@Param() urlParams: TaskUrlParams, @CurrentUser() currentUser: ICurrentUser): Promise<TaskResponse> {
+		const taskWithSatusVo = await this.taskUc.find(currentUser.userId, urlParams.taskId);
+
+		const response = TaskMapper.mapToResponse(taskWithSatusVo);
+		return response;
+	}
+
+	@Post()
+	async create(@Body() params: TaskCreateParams, @CurrentUser() currentUser: ICurrentUser): Promise<TaskResponse> {
+		const taskWithSatusVo = await this.taskUc.create(currentUser.userId, params.courseId);
+
+		const response = TaskMapper.mapToResponse(taskWithSatusVo);
+		return response;
+	}
+
+	@Patch(':taskId')
+	async update(
```

Comment:
This looks like the use case is "update an individual task". What exactly is this UC about? That should be then implemented in the UC method. Also `ITaskUpdate` looks very generic. We might have different use cases to update a task in the future.

Reply:
for now, this case for creating / editing / find a task is the only uc
it might sound special for "individual assignment task", but there is *so far* no distinction between a normal task, and one which can be assigned to individuals